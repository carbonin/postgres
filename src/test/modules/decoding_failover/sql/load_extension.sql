CREATE EXTENSION decoding_failover;

SELECT decoding_failover_create_logical_slot('test_slot', 'test_decoding');

SELECT decoding_failover_advance_logical_slot('test_slot', txid_current(), txid_current(), pg_current_xlog_location(), pg_current_xlog_location());

SELECT pg_drop_replication_slot('test_slot');
