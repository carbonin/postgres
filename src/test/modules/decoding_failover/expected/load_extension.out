CREATE EXTENSION decoding_failover;
SELECT decoding_failover_create_logical_slot('test_slot', 'test_decoding');
 decoding_failover_create_logical_slot 
---------------------------------------
 
(1 row)

SELECT decoding_failover_advance_logical_slot('test_slot', txid_current(), txid_current(), pg_current_xlog_location(), pg_current_xlog_location());
 decoding_failover_advance_logical_slot 
----------------------------------------
 
(1 row)

SELECT pg_drop_replication_slot('test_slot');
 pg_drop_replication_slot 
--------------------------
 
(1 row)

